src/main/java/com/foundation/salesforce/entities/Lead.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * @(#) Lead.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.entities;
+
+/**
+ * lead class which saves relevant lead data.
```

Comment:
Please change the beginning of the comment, this always begins with the first capital letter.

Reply:
done